cmd/apiserver-boot/boot/create_resource.go

```diff
@@ -183,6 +186,8 @@ import (
 )
 
 // +genclient=true
+{{- if .NonNamespacedKind }}
+// +nonNamespaced=true{{ end }}
```

Comment:
That all sounds good.

> What do you think will be the right thing to do here?
> Maybe create a xxx_strategy.go file with the custom strategy implementation stub when the "--non-namespaced" flag is present on the apiserver-boot create-resource command line?

create-resource already updates the custom strategy with an empty validation function: `func ({{.Kind}}Strategy) Validate(ctx request.Context, obj runtime.Object)`

Why not put it just following that?